sdk/python/feast/type_map.py

```diff
@@ -444,10 +444,14 @@ def pa_to_feast_value_type(value: object) -> ValueType:
         "list<item: binary>": ValueType.BYTES_LIST,
         "list<item: bool>": ValueType.BOOL_LIST,
     }
-    return type_map[value.type.__str__()]
+
+    if isinstance(value, pa.lib.ChunkedArray):
```

Comment:
Why isn't this check done in the first line (or even in the argument as a mypy type)?

Reply:
Ah okay, because it is only getting used in 1 place, I have moved the `pa.lib.ChunkedArray` to the arguments. Have rebased the commit